week-3/A-array-find/exercise.js

```diff
@@ -5,7 +5,9 @@
 */
 
 // -- Complete this function -->
-function findLongNameThatStartsWithA(names) {}
+function findLongNameThatStartsWithA(name) {
+  return name[0] === "A" && name.length > 7;
```

Comment:
I didn't know this would work, but it does :slightly_smiling_face: . Don't forget that `name` is a string though,not an array.